### PR TITLE
Fix misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: EM_FHPath.py [hunk 1]
```diff
@@ -259,7 +259,7 @@ def execute(self, obj):
             self.ww = [Vector(0,0,0)]
         else:
             # transform 'obj.ww' according to the 'Base' Placement
-            # (transation is don't care, we worry about rotation)
+            # (translation is don't care, we worry about rotation)
```
Comment (luzpaz) [inline EM_FHPath.py:257]: is this correct or should it be `transaction` ?

**Comment by luzpaz** [comment]
bump

File: EM_FHPath.py (hunk 1 above)
Comment (ediloren) [reply EM_FHPath.py:257]: correct thx

**Comment by luzpaz** [comment]
Rebased, please merge @ediloren  
Thanks

**Pull request merged**